libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -146,13 +144,12 @@ public void onSaveInstanceState(Bundle outState) {
    * @param savedInstanceState to extract state variables
    */
   public void onRestoreInstanceState(Bundle savedInstanceState) {
-    boolean isVisible = savedInstanceState.getBoolean(getContext().getString(R.string.recenter_btn_visible));
-    recenterBtn.setVisibility(isVisible ? View.VISIBLE : View.INVISIBLE);
-    int bottomSheetState = savedInstanceState.getInt(getContext().getString(R.string.bottom_sheet_state));
-    resetBottomSheetState(bottomSheetState);
-    boolean instructionViewVisible = savedInstanceState.getBoolean(getContext().getString(
-      R.string.instruction_view_visible));
-    updateInstructionListState(instructionViewVisible);
+    NavigationViewInstanceState navigationViewInstanceState = savedInstanceState.getParcelable(
```

Comment:
NIT - What about splitting up the call as follows?
```java
NavigationViewInstanceState navigationViewInstanceState = savedInstanceState
  .getParcelable(getContext().getString(R.string.instance_state));
```
I believe it reads better.